<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_decompress">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>450</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Decompress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_file">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>450</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>640</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
